### PR TITLE
PAYEE-321: Upgraded view to new frontend library

### PR DESCRIPTION
**Pull request by Jamie22Butler**
- [x] Unit tests passing
- [ ] Coverage reached
- [ ] Acceptance tests passing
- [x] Screen Reader checks 
- [ ] Reviewed

### CODE REVIEW

File: app/views/incomes/YourIncomeCalculationView.scala.html [hunk 1]
```diff
@@ -32,101 +32,104 @@
...
-                                <tr>
-                                    <th>@Messages("tai.income.calculation.incomeTable.dateHeader")</th>
-                                    <th class="text--right">@Messages("tai.income.calculation.incomeTable.incomeHeader")</th>
-                                    <th class="text--right">@Messages("tai.income.calculation.incomeTable.taxPaidHeader")</th>
-                                    <th class="text--right">@Messages("tai.income.calculation.incomeTable.nationalInsuranceHeader")</th>
+                        <table id="taxable-income-table" class="govuk-table">
+                            <caption class="text--left table__caption table__caption--bottom-border govuk-visually-hidden">@Messages("tai.income.calculation.incomeTable.caption")</caption>
+                            <thead class="govuk-table__head">
+                                <tr class="govuk-table__row">
+                                    <th scope="col" class="govuk-table__header">@Messages("tai.income.calculation.incomeTable.dateHeader")</th>
```
Comment (sergi-hmrc) [inline app/views/incomes/YourIncomeCalculationView.scala.html:104]: are the scopes really needed?

Comment (Jamie22Butler) [reply app/views/incomes/YourIncomeCalculationView.scala.html:104]: Going off of the gds pattern they are. Up for debate; however, as to whether they're required - passes HTML Validator check :)

Comment (masahmad) [reply app/views/incomes/YourIncomeCalculationView.scala.html:104]: yep needed  for screen readers

File: app/views/incomes/YourIncomeCalculationView.scala.html [hunk 2]
```diff
@@ -32,101 +32,104 @@
...
     authedUser = Some(user),
-    pageHeader = Some(header),
-    employerName = Some(model.employerName),
-    articleClasses = None
+    pageTitle = Some(Messages("tai.service.navTitle", model.employerName)),
+    backLinkUrl = Some(routes.TaxAccountSummaryController.onPageLoad.url),
+    backLinkContent = Messages("tai.back-link.upper"),
+    backLinkId = "backLink",
+    pagePrintable = true,
+    pagePrintName = Some("your-income-calculation-view")
```
[the remark this answers is not in the document]
Comment (Jamie22Butler) [reply app/views/incomes/YourIncomeCalculationView.scala.html:52]: Gotcha, will remove it now.

**Review by masahmad — APPROVED** [review]
LGTM

**Review by sergi-hmrc — APPROVED** [review]

**Pull request merged**